src/plugins/discover/tsconfig.json

```diff
@@ -52,6 +52,7 @@
     "@kbn/rison",
     "@kbn/config-schema",
     "@kbn/storybook",
+    "@kbn/visualizations-plugin",
```

Comment:
This was to have a type for the "saved search" SO type. Not sure if there is a better alternative. :)

Reply:
See https://github.com/elastic/kibana/pull/150631#discussion_r1105249324 for what I'm thinking.